content/docs/studio/index.md

```diff
@@ -20,5 +20,5 @@ Use DVC, CML and Studio to:
 
 - **Use the power of Git to track and preserve all your experiments.**
 
-- **Run CI/CD for your ML projects on cloud resources of your choice without any
-  new tools.**
+- **Run [CI/CD](https://en.wikipedia.org/wiki/CI/CD) for your ML projects on
```

Comment:
Needed?

Reply:
In #2468 there is a checkbox that reads "The CI/CD part may need more explaining/linking" and the easiest way to link was Wikipedia. 😅  We can replace it with a concept page sometime I think.

Reply:
Yes, it's a good idea for now. I meant explaining CML runners I guess, or something related with CML. But not for this iteration. Anyway, we'll review when possible.